dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -156,6 +158,22 @@ public static void registerZenodoDOIForWorkflow(ApiClient zenodoClient, String d
 
         Deposit publishedDeposit = publishDepositOnZenodo(actionsApi, depositionID);
 
+        Object links = publishedDeposit.getLinks();
+        String conceptDoiUrl = (String)((LinkedHashMap)links).get("conceptdoi");
+        String conceptDoi = conceptDoiUrl;
```

Comment:
Can you move this into a separate function that takes in conceptDoiUrl and outputs conceptDoi, then write a small unit test for it?

Reply:
Created a new function extractDoiFromDoiUrl and will create a test for it

Reply:
test created io.dockstore.webservice.helpers.ZenodoHelperTest#extractDoiFromDoiUrl